Cargo.toml

```diff
@@ -3,8 +3,10 @@ members = [
     "bench-vortex",
     "encodings/*",
     "fuzz",
+    "geo",
```

Comment:
Not `vortex-geo`?

Reply:
maybe we can do this like `encodings/` where we have `extensions/geo` and then the crate type is `vortex-geo`